docs/boshdirector.adoc

```diff
@@ -1,20 +1,32 @@
-= Installing Bosh Director
+= Configuring Ops Manager Director
 Eitan Suez <esuez@pivotal.io>
 v0.1, 2017
 :project_id: {{project_id}}
 :domain_name: {{domain_name}}
 :env_name: {{env_name}}
 :director_ip_address: {{director_ip_address}}
+:region: {{region}}
 
+Estimated time:  30 minutes
+
+= Requirements
+
+Lab Requirements
+
+= What you will learn
+
+* How to configure and install Ops Manager Director
```

Comment:
Perhaps in their current state, yes we could remove them - particularly in this case where it is identical to the title of the lab.  But stating the purpose of the lab is not in itself a bad idea I think.